buildSrc/src/main/groovy/io/deephaven/tools/docker/WaitForHealthyContainer.groovy

```diff
@@ -0,0 +1,66 @@
+package io.deephaven.tools.docker
+
+import com.bmuschko.gradle.docker.tasks.container.DockerExistingContainer
+import com.github.dockerjava.api.command.InspectContainerCmd
+import com.github.dockerjava.api.command.InspectContainerResponse
+import groovy.transform.CompileStatic
+import org.gradle.api.GradleException
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Optional
+
+@CompileStatic
+class WaitForHealthyContainer extends DockerExistingContainer {
```

Comment:
This is copied wholesale from the gradle plugin we're using, since they have it marked for removal - they consider it to be too specific of a script to include in the plugin itself